_sass/modules/_footer.scss

```diff
@@ -236,32 +218,70 @@ footer {
     &::before {
       content: '';
       display: inline-block;
-      width: $icon-size--s;
-      height: $icon-size--s;
+      width: $footer-icon-size;
+      height: $footer-icon-size;
       vertical-align: middle;
       margin: 0 12px 0 0;
-      opacity: .38;
+      border-radius: 50%;
+      border: 2px solid rgba($icon-border-color, .2);
     }
   }
 
   .icon-github {
-    @include social-icon('../img/icon_github.svg');
-  }
-
-  .icon-google {
-    @include social-icon('../img/icon_google-plus.svg');
+    @include social-icon('../img/icons/icon-github.svg');
   }
 
   .icon-twitter {
-    @include social-icon('../img/icon_twitter.svg');
+    @include social-icon('../img/icons/icon-twitter.svg');
   }
 
   .icon-stack-overflow {
-    @include social-icon('../img/icon_stack-overflow.svg');
+    @include social-icon('../img/icons/icon-stack-overflow.svg');
   }
 
   .icon-youtube {
-    @include social-icon('../img/icon_youtube.svg');
+    @include social-icon('../img/icons/icon-youtube.svg');
```

Comment:
These links `icon-stack-overflow `  and `icon-youtube` are commented. Do we need them in the future? If not remove HTML images and styles, please.

Reply:
These icons will be used in the future.